config/dbConnection.js

```diff
@@ -2,7 +2,7 @@ const { connection, connect } = require("mongoose")
 
 function db () {
     //setUp of variables
-    const uri = process.env.DB_URI
+    const uri = process.env.DB_URI || "mongodb://127.0.0.1:27017/sharedBox"
```

Comment:
There are duplicated changes from #2. Remember, before creating a new branch move to `dev` branch and the pull changes from the remote to update your local branch. After that, you can create your new branch.